Casks/t/timche-gmail-desktop.rb

```diff
@@ -7,6 +7,8 @@
   desc "Unofficial Gmail desktop app"
   homepage "https://github.com/timche/gmail-desktop"
 
+  disable! date: "2025-05-09", because: :discontinued, replacement_cask: "meru"
```

Comment:
```suggestion
  deprecate! date: "2025-05-09", because: :discontinued, replacement_cask: "meru"
```

We should deprecate the cask first. It will then be disabled after one year.